frontend/src/SettingsView/BillingView/constants/stripe.js

```diff
@@ -1,6 +1,6 @@
 const STRIPE_PUBLISHABLE =
 	process.env.NODE_ENV === 'production'
-		? 'pk_live_MY_PUBLISHABLE_KEY'
+		? 'pk_live_GVMtxntEIujtyI7ZOBA45dQG'
```

Comment:
y'all will want to just use the test key across the board during the development phase of the application, otherwise you'll ring up a bunch of charges against whoever's Stripe account the live key is associated with.